Pipeline_example/scripts/exceptions.py

```diff
@@ -0,0 +1,10 @@
+"""
+Taken from Jenna's script from the Cellpainting repo
+This class defines a custom exception class for exceeding the max workers on a machine.
+"""
+
+class MaxWorkerError(Exception):
+    """
+    Raised when the number of workers assigned to `max_workers` exceeds the number of CPU/workers on the machine. 
+    """
+    pass
```

Comment:
Add new endline to this file to remove error. Essentially what this means is the file is 10 lines of code but it needs one line at the bottom at is empty so there is a total of 11 lines.

Reply:
Got it, thanks!